front/app/component-library/components/CollapsibleContainer/index.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { ReactNode, useEffect, useState } from 'react';
```

Comment:
This component shares a lot of similarities with the Accordion, but I decided to keep them separate and just duplicate some of the code to make it cleaner.